test/evergreen.yml

```diff
@@ -1027,6 +1027,16 @@ tasks:
       - func: "fetch artifacts"
       - func: "unit test"
 
+  - name: unit-test-long
+    tags: ["unit_test"]
+    depends_on:
+    - name: compile
+    commands:
+      - func: "fetch artifacts"
+      - func: "unit test"
+        vars:
+          unit_test_args: ${unit_test_args} --long
```

Comment:
Feel a bit over-engineered. `-v 2 --long` is fine to me.

Reply:
Noted.